_posts/shwetashambhavi/2020-10-21-dictionary-pair-programming-shwetashambhavi.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+author: shwetashambbhavi
+author: EAkita
```

Comment:
This will need to be
```
author:
- author1
- author2
```
That's how YAML sees a list of things. Right now you're setting author equal to one thing, then another.

Reply:
got it.. changed it to as specified